compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/error.ref-value-in-event-handler-wrapper.tsx

```diff
@@ -0,0 +1,28 @@
+// @enableInferEventHandlers
+// @validateRefAccessDuringRender
```

Comment:
just noticed this - all the @ pragmas need to be on the first line (also the same fix in other files). I think the reason you're getting errors is that validateRefAccessDuringRender is on by default